examples/ADC Flow Sample.json

```diff
@@ -0,0 +1,100 @@
+[
+    {
+        "id": "8cdbf6a7f8e518a8",
+        "type": "inject",
+        "z": "aa7eb918109cfafe",
+        "name": "",
+        "props": [
+            {
+                "p": "readType",
+                "v": "single",
+                "vt": "str"
+            },
+            {
+                "p": "adc",
+                "v": "2",
+                "vt": "str"
+            },
+            {
+                "p": "payload"
+            },
+            {
+                "p": "dataRate",
+                "v": "800",
+                "vt": "str"
+            }
+        ],
+        "repeat": "",
+        "crontab": "",
+        "once": false,
+        "onceDelay": "1",
+        "topic": "",
+        "payload": "3",
+        "payloadType": "num",
+        "x": 90,
+        "y": 180,
+        "wires": [
+            [
+                "3a5834a3ade508b4"
+            ]
+        ]
+    },
+    {
+        "id": "21a4336c1fe5f760",
+        "type": "debug",
+        "z": "aa7eb918109cfafe",
+        "name": "debug adc output",
+        "active": true,
+        "tosidebar": true,
+        "console": false,
+        "tostatus": false,
+        "complete": "payload",
+        "targetType": "msg",
+        "statusVal": "",
+        "statusType": "auto",
+        "x": 700,
+        "y": 140,
+        "wires": []
+    },
+    {
+        "id": "3a5834a3ade508b4",
+        "type": "adc",
+        "z": "aa7eb918109cfafe",
+        "name": "",
+        "transport": "Local",
+        "tcpAddress": "192.168.1.193",
```

Comment:
do we need tcp address value?

Reply:
The default transport is Local, so the tcpAddress wouldn't show up. I left the IP there so the user knows what to input. 
I have removed it.